src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -54,11 +54,13 @@ Copyright (c) .NET Foundation. All rights reserved.
    -->
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and
                             '$(HasRuntimeOutput)' == 'true' and
-                            $([MSBuild]::IsOSPlatform(`Windows`))and
+                            $([MSBuild]::IsOSPlatform(`Windows`)) and
                             '$(RuntimeIdentifier)' == ''">
     <_UsingDefaultRuntimeIdentifier>true</_UsingDefaultRuntimeIdentifier>
     <RuntimeIdentifier Condition="'$(PlatformTarget)' == 'x64'">win7-x64</RuntimeIdentifier>
     <RuntimeIdentifier Condition="'$(PlatformTarget)' == 'x86' or '$(PlatformTarget)' == ''">win7-x86</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="'$(PlatformTarget)' == 'x64' and $([MSBuild]::AreFeaturesEnabled('17.10'))">win-x64</RuntimeIdentifier>
```

Comment:
I would suggest adding an opt out environment variable here such as DOTNET_CLI_DISABLE_SIMPLE_RID_IMPLY, since every time I have made even the smallest change to this file, there is someone out there with a weird custom logic solution that will be broken 💦

Reply:
(Make an msbuild property instead of an environment variable if possible, its possible people would want the rest of the change wave but will want this to be disabled, IMO)

Reply:
Also please retarget to 9.0.